site/app/controllers/forum/ForumController.php

```diff
@@ -686,8 +740,14 @@ public function alterPost($modify_type) {
             $content = "In " . $full_course_name . "\n\nThread: " . $thread_title . "\n\nPost:\n" . $post["content"] . " was deleted.";
             $event = [ 'component' => 'forum', 'metadata' => $metadata, 'content' => $content, 'subject' => $subject, 'recipient' => $post_author_id, 'preference' => 'all_modifications_forum'];
             $this->core->getNotificationFactory()->onPostModified($event);
-
             $this->core->getQueries()->removeNotificationsPost($post_id);
+            if ($type === "thread") {
+                $this->sendSocketMessage(['type' => 'delete_thread', 'thread_id' => $thread_id]);
+            }
+            if ($type === "post") {
+                $post_id = $_POST["post_id"];
+                $this->sendSocketMessage(['type' => 'delete_post', 'thread_id' => $thread_id, 'post_id' => $post_id]);
+            }
```

Comment:
Same here

Reply:
Fixed in https://github.com/Submitty/Submitty/pull/9906/commits/0abd2b762dd9d9563deb417f91b583ef6d165da9